core/src/main/java/com/netflix/conductor/dao/MetadataDAO.java

```diff
@@ -62,73 +55,34 @@
     /**
      * @param def workflow definition
      */
-    void create(WorkflowDef def);
+    void createWorkflowDef(WorkflowDef def);
```

Comment:
Same as above.

Reply:
same as above, non-200 for failure and 200 for success.